xds/internal/client/v2client_ack_test.go

```diff
@@ -237,15 +238,16 @@ func (s) TestV2ClientAckNackAfterNewWatch(t *testing.T) {
 	t.Log("Started xds v2Client...")
 
 	// Start the watch, send a good response, and check for ack.
-	cbLDS := startXDS(t, "LDS", v2c, fakeServer.XDSRequestChan, goodLDSRequest)
-	sendGoodResp(t, "LDS", fakeServer, versionLDS, goodLDSResponse1, goodLDSRequest, cbLDS)
+	cbLDS := startXDS(t, "LDS", v2c, fakeServer.XDSRequestChan, goodLDSRequest, "", "")
+	nonce := sendGoodResp(t, "LDS", fakeServer, versionLDS, goodLDSResponse1, goodLDSRequest, cbLDS)
 	versionLDS++
```

Comment:
NIt: How about moving the `versionLDS++` line to after the call to `startXDS`. That way, we wont have to do the `versionLDS-1`.

Reply:
Done